Floofbot/Services/EventLoggerService.cs

```diff
@@ -56,7 +64,34 @@ public bool IsToggled(IGuild guild)
 
             return ServerConfig.IsOn;
         }
-
+        private async Task HandleBadMessage(SocketUser user, SocketMessage msg)
+        {
+            await msg.DeleteAsync();
+            var botMsg = await msg.Channel.SendMessageAsync($"{user.Mention} watch your language! You've said a bad word!");
+            Thread.Sleep(5000);
```

Comment:
Judging by the linked SO question (and previous code in our codebase) I think we should go with `await Task.Delay(5000)` instead of `Thread.Sleep(5000)`, since the former goes along with the async-await architecture that we already have. 
https://stackoverflow.com/questions/20082221/when-to-use-task-delay-when-to-use-thread-sleep

Other than this, lgtm.